docs/conf.py

```diff
@@ -39,12 +39,14 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    "m2r",  # Markdown sphinx docs support
+    "sphinx.ext.napoleon",  # numpy style doc strings for autodoc
     "sphinx.ext.autodoc",
     "sphinx.ext.doctest",
-    "sphinx.ext.todo",
+    # "sphinx.ext.todo",
```

Comment:
i'm really not sure what these extensions are doing so commenting out for the time being to slim things down.